### ISSUE TITLE
Cannot read stream of json objects

### ISSUE THREAD

**Issue by Terramorpha** [aconchillo/guile-json#76]
I tried using this library to read the output of a program which prints a stream of json objects. However, the reader won't simply consume the first object of a stream.

The following snippet should, but doesn't work
```scheme
(use-modules (json))
(with-input-from-string "{} {}" json->scm)
```

**Comment by Terramorpha** [comment]
Oops, never mind, Guix's version was outdated. Sorry.

**Comment by Terramorpha** [comment]
Actually, the new `json-seq->scm` doesn't fit my usecase either. It would be great if `json->scm` didn't check further than the end of the first object it finds.

**Comment by aconchillo** [comment]
Hi @Terramorpha! Sorry for the delay. There are multiple ways to parse a stream of JSON documents, see https://en.wikipedia.org/wiki/JSON_streaming.

guile-json supported JSON Text Sequences (not in any release yet). I have now added support for concatenated JSON. You need to pass `#concatenated #t` to `(json->scm)`. With this, the parse won't fail and you will be able to keep calling `(json-scm)` on the same port as long as valid documents are parsed.

Let me know if this works for you. Thanks!

**Comment by aconchillo** [comment]
This is now available in guile-json 4.6.0.
